tpgtools/handwritten.go

```diff
@@ -74,7 +74,7 @@ func copyHandwrittenFiles(inPath string, outPath string) {
 		}
 
 		// Write copied file.
-		err = ioutil.WriteFile(path.Join(outPath, terraformResourceDirectory, f.Name()), b, 0644)
+		err = ioutil.WriteFile(path.Join(outPath, terraformResourceDirectory, "tpgresource", f.Name()), b, 0644)
```

Comment:
Could we put these helpers in a separate folder? That would make it easier for us to remove these helpers later.

Reply:
Will move the DCL helper files to the tpgdclresource folder.

Reply:
Done